### DISCUSSION TITLE
Seems like an error / mislabel

### DISCUSSION THREAD

**Discussion by DoctorEam** [lmusarella/Pancakeswap-Prediction-Bot-2023#16] [General]
![Screenshot from 2023-06-10 11-24-56](https://github.com/lmusarella/Pancakeswap-Prediction-Bot/assets/38581667/8dc29116-bb13-4b76-bea7-56a77a367e5c)
![Screenshot from 2023-06-10 11-25-06](https://github.com/lmusarella/Pancakeswap-Prediction-Bot/assets/38581667/2f8f5516-9a13-4390-bd4e-9fd3a4f785d1)

There seems to be an error / mislabel with the quote reading. It assigns the quote of the next round to the previous one. 
(Simulation mode!)

**Comment by DoctorEam** [comment]
I guess simulation mode is not my friend. And autoclaim could be a problem in simulation mode.

**Comment by lmusarella** [comment]
> ![Screenshot from 2023-06-10 11-24-56](https://github.com/lmusarella/Pancakeswap-Prediction-Bot/assets/38581667/8dc29116-bb13-4b76-bea7-56a77a367e5c)
> ![Screenshot from 2023-06-10 11-25-06](https://github.com/lmusarella/Pancakeswap-Prediction-Bot/assets/38581667/2f8f5516-9a13-4390-bd4e-9fd3a4f785d1)
> 
> There seems to be an error / mislabel with the quote reading. It assigns the quote of the next round to the previous one. 
> (Simulation mode!)

hello, then most likely the odds are right. The problem with the quota strategy is that the majority of users use bots. And bots usually always place the bet 10 seconds before, so the odds are likely to change within 2 seconds.
